src/pages/junghyowon/Login/JungLogin.js

```diff
@@ -1,7 +1,103 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './JungLogin.scss';
 
 function JungLogin() {
-  return <div>정효원 로그인페이지</div>;
+  const [id, setId] = useState('');
+  const [password, setPw] = useState('');
+  const navigate = useNavigate();
+  const idPw = id.indexOf('@') !== -1 && password.length > '5';
```

Comment:
```suggestion
  const idPw = id.includes('@') && password.length > '5';
```
- includes()를 사용하는 방법도 있습니다.